compiler-rt/lib/hwasan/hwasan_linux.cpp

```diff
@@ -561,4 +542,25 @@ void InstallAtExitCheckLeaks() {
 
 }  // namespace __hwasan
 
+using namespace __hwasan;
```

Comment:
Is this only for `hwasanThreadList` and `BlockSignals`?

Reply:
I believe it is also for `Thread` and `GetCurrentThread()`.